media/json-jackson/pom.xml

```diff
@@ -96,6 +96,11 @@
             <artifactId>jersey-common</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
```

Comment:
This does not sound right. Jackson being used on the client side surely does not need jersey-server.